yarn2nix.nix

```diff
@@ -5,11 +5,6 @@ let
     url = "https://github.com/moretea/yarn2nix/archive/v1.0.0.tar.gz";
     sha256 = "02bzr9j83i1064r1r34cn74z7ccb84qb5iaivwdplaykyyydl1k8";
   }) { inherit pkgs nodejs; };
-  networkMap = {
-    mainnet = "mainnet";
-    staging = "testnet";
```

Comment:
@rvl is this a joke? :)
I suppose this line should read:
```
staging = "staging";
```

Reply:
Oh sorry for the fake alarm... I just saw that these lines are deleted all together 🤦‍♂️

Reply:
Hehe yes I was happy to delete that.